### ISSUE TITLE
System.TypeLoadException: VTable setup of type XF.Material.Forms.UI.MaterialButton failed

### ISSUE THREAD

**Issue by AnthonyLatty** [contrix09/XF-Material-Library#57]
**Describe the bug**
I honestly dont know where this error came from but it all happened when i was preparing an app for production. Doing the steps: Setting the build option to release mode etc. It throws this error `System.TypeLoadException: VTable setup of type XF.Material.Forms.UI.MaterialButton failed`
I have AppCenter logging my crashes and the error it reports is: 
`    xxxx.Views.Audio.xxxxx.DetailsPage..ctor (Testing.Models.Audio audio) [0x00008] in <dd419e9e473f46438506fd55f81978c1>:0
    AudioPage+<AudioListView_OnItemTapped>d__5.MoveNext ()
    AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state)
    SyncContext+<>c__DisplayClass2_0.<Post>b__0 ()
    Thread+RunnableImplementor.Run ()
    IRunnableInvoker.n_Run (System.IntPtr jnienv, System.IntPtr native__this)
    (wrapper dynamic-method) System.Object.51(intptr,intptr)`

**To Reproduce**
1. Download latest plugin version
2. Set build action to release mode

**Expected behavior**
App will crash and display `System.TypeLoadException: VTable setup of type XF.Material.Forms.UI.MaterialButton failed`

**Desktop (please complete the following information):**
 - OS: Mac OS Mojave
 - Browser Firefox

**Smartphone (please complete the following information):**
 - Device: Honor 6x
 - OS: Android Pie 9.0
 - Browser stock browser 
 - Version 9.0

**Additional context**
To by pass this bug, i downgrade the version of this plugin to v1.3.0.5. This version was the last stable version for me hence thats why i downgraded to it.

**Comment by contrix09** [comment]
Hi, for now the target Android version of the library is still 8.1. But I will still check if this issue still occur on when I update the version.

**Comment by muhammet-mucahit** [comment]
Any solution? I have the same trouble with Mac OS Mojave, target iOS.

**Comment by contrix09** [comment]
Hmm, this is weird. Can you provide a list of Nuget packages you are using?

**Comment by contrix09** [comment]
Hi, can u download the new version `1.4.0.0` and see if this issue still exists.

**Comment by muhammet-mucahit** [comment]
Sorry for the late response. I am trying to use new version but I have a lot of problems with the new version. 

I use these Nuget packages:
- Xamarin.Forms (3.5.0.169047)
- XF.Material (1.4.1.0)

I am getting error in XAML file:
- No embedded resource found XF.Material.Forms.Resources.MaterialColors

Please help me about that errors. I use Visual Studio for Mac.

**Comment by muhammet-mucahit** [comment]
This issue for MaterialButton is okay. But as I mentioned above, there is a new problem with the new versions. I can use MaterialButton with 1.3.1.1. Otherwise I am getting error in MainPage.xaml

- No embedded resource found XF.Material.Forms.Resources.MaterialColors

When I run my app everything is working well, the problem is only in rendering of preview in xaml file.

Best of Luck

**Comment by AnthonyLatty** [comment]
When i am updating the application again i will test the new version of this plugin, as i stated i downgraded the version of the plugin. I cant give a valid response if its working or not until i do a next update in the future.

**Comment by AnthonyLatty** [comment]
what is the Target Android version for the latest version of this plugin? @contrix09

**Comment by contrix09** [comment]
The target Android version is nov `v9.0`.

**Comment by AnthonyLatty** [comment]
There seems to be an error after upgrading, in the Android section MainActivity.cs where the plugin is instantiated with `XF.Material.Droid.Material.Init(this, savedInstanceState);` before `LoadApplication(new App());` it throws an error when compiling saying `The type or namespace name Droid does not exist in the namespace`

**Comment by contrix09** [comment]
Even when the `XF.Material` package was installed in the Android project? Can you try to delete the `bin` and `obj` folders.

**Comment by AnthonyLatty** [comment]
yes

**Comment by contrix09** [comment]
Hi, I tried installing the package to an empty project and encountered the problem above. The installed Android project template in VS for Xamarin.Forms defaults to target framework version of `v8.1`. I was not able to find the `XF.Material.Droid` namespace. I tried to update the pre-installed `Xamarin.Android.Support.*` packages to version `28.0.0.*` and found some conflicts. What I did was uninstalled the packages referencing the `Xamarin.Android.Support.*` version `27.0.0.*`, then uninstalled the `Xamarin.Android.Support.*` version `27.0.0.*`. 

So basically I uninstalled all nuget packages in the Android project from an empty Xamarin.Forms project. Then installed `Xamarin.Forms` then `XF.Material`. No need to install again the `Xamarin.Android.Support.*` packages since `Xamarin.Forms` and `XF.Material` installs them automatically.

Here is the sample empty project: [Sample.zip](https://github.com/contrix09/XF-Material-Library/files/2898849/Sample.zip)
